x-pack/plugins/synthetics/public/apps/synthetics/state/synthetics_enablement/index.ts

```diff
@@ -58,6 +58,7 @@ export const syntheticsEnablementReducer = createReducer(initialState, (builder)
         areApiKeysEnabled: state.enablement?.areApiKeysEnabled ?? false,
         canManageApiKeys: state.enablement?.canManageApiKeys ?? false,
         isEnabled: false,
+        isValidApiKey: true,
```

Comment:
It feels a *little* weird to have this set to true by when the synthtics key is deleted, but I understand that it's based on whether or not the api key is in an error. I might prefer to see this set to null when the key is deleted, but I'm not terribly opinionated